core/src/test/java/org/infinispan/scattered/statetransfer/CoordStopTest.java

```diff
@@ -0,0 +1,166 @@
+package org.infinispan.scattered.statetransfer;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.Future;
+import java.util.concurrent.TimeUnit;
+
+import org.infinispan.commands.ReplicableCommand;
+import org.infinispan.configuration.cache.BiasAcquisition;
+import org.infinispan.configuration.cache.CacheMode;
+import org.infinispan.configuration.cache.ConfigurationBuilder;
+import org.infinispan.configuration.global.GlobalConfigurationBuilder;
+import org.infinispan.distribution.MagicKey;
+import org.infinispan.notifications.Listener;
+import org.infinispan.notifications.cachelistener.annotation.TopologyChanged;
+import org.infinispan.notifications.cachelistener.event.TopologyChangedEvent;
+import org.infinispan.remoting.transport.Address;
+import org.infinispan.statetransfer.StateRequestCommand;
+import org.infinispan.statetransfer.StateResponseCommand;
+import org.infinispan.test.MultipleCacheManagersTest;
+import org.infinispan.test.fwk.CleanupAfterMethod;
+import org.infinispan.test.transport.DelayedViewJGroupsTransport;
+import org.infinispan.topology.CacheTopology;
+import org.infinispan.util.BlockingLocalTopologyManager;
+import org.infinispan.util.BlockingLocalTopologyManager.BlockedTopology;
+import org.infinispan.util.ControlledConsistentHashFactory;
+import org.infinispan.util.ControlledRpcManager;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.Test;
+
+@Test(groups = "functional", testName = "scattered.statetransfer.CoordStopTest")
+@CleanupAfterMethod
+public class CoordStopTest extends MultipleCacheManagersTest {
```

Comment:
We actually call it "coordinator" ;)

Reply:
Personal question: are you Dan or Daniel on your birth certificate? :)

Reply:
Dan :smile: 

I know some of our test names are ridiculously long, but I don't think `CoordinatorStopTest` would be one of them ;)